.github/workflows/coverage.yml

```diff
@@ -6,6 +6,8 @@ on:
     - cron: "0 0 * * *"
   push:
     branches: [ dev ]
+  pull_requst:
+    branches: [ '**' ]
```

Comment:
Am I right that we do this temporarily so we see the results in this PR right away?